test/CMakeLists.txt

```diff
@@ -1 +1,6 @@
 add_subdirectory(api)
+add_subdirectory(shaping)
+# Not ready yet
+if (NOT WIN32)
+  add_subdirectory(fuzzing)
```

Comment:
This should be checking for MSVC, the fuzzing is a clang/gcc feature.

Reply:
Done.